tests/data/PBS-gnu-and-musl.toml

```diff
@@ -0,0 +1,63 @@
+[lift]
+name = "gnu-and-musl"
+description = "Tests a Python scie that can run on either a gnu or musl libc Linux machine."
```

Comment:
@sureshjoshi not really sure if this is of interest to you. Had a fellow on discord that needed it though - a ~static Linux Python binary that can run on both gnu & musl Linux. As long as your Python deps work on a statically linked Python interpreter, this demonstrates how you can now do it.

Reply:
This is great, thanks! I run Alpine linux a lot around here - and I recall running into an issue (a long time ago) with some tool I packaged up. I think it may have been musl-related, but have to check again